src/commands/title-escrow/endorseTransferOfOwner.ts

```diff
@@ -0,0 +1,56 @@
+import { Argv } from "yargs";
+import { error, info, success, warn } from "signale";
+import { getLogger } from "../../logger";
+import { endorseTransferOfOwner } from "../../implementations/title-escrow/endorseTransferOfOwner";
+import { BaseTitleEscrowCommand as TitleEscrowEndorseTransferOfOwnerCommand } from "../../commands/title-escrow/title-escrow-command.type";
+import { withGasPriceOption, withNetworkAndWalletSignerOption } from "../shared";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("title-escrow:endorse-transfer-of-owner");
+
+export const command = "endorse-transfer-owner [options]";
+
+export const describe =
+  "Endorses the transfer of owner of transferable record to an approved owner and approved holder address";
```

Comment:
might want to pick one terminology here and stick with it - either nominated or approved

Reply:
i'll probably stick to approved terminology